magefile.go

```diff
@@ -948,6 +959,64 @@ func (d Dev) deployAPIMock(ctx context.Context, cluster *dev.Cluster) error {
 	return nil
 }
 
+func deployFeatureToggles(ctx context.Context, cluster *dev.Cluster) error {
+	availableFeatureToggles := []featuretoggle.TestableFeatureToggleHandler{
+		&featuretoggle.MonitoringStackFeatureToggle{
+			Client:         cluster.CtrlClient,
+			SchemeToUpdate: cluster.Scheme,
+		},
+	}
+
+	for _, featTog := range availableFeatureToggles {
+		// feature toggles enabled/disabled at the level of openshift/release in the form of multiple jobs
+		if featTog.IsEnabledOnTestEnv() {
+			if err := featTog.Enable(ctx); err != nil {
```

Comment:
That's what the purpose of `Enable` method is: to do whatever it takes to actually enable the feature toggle, which at the moment for monitoringStackFeatureToggle, is to update/create the AddonOperator CR with the right spec containing `spec.featureToggles.ExperimentalFeatures: true` .

Although that might seem straightforward at the moment, in theory, there can be more nuanced feature toggles in the future, which might not be as straightforward to enable and say, might require some pre-requisite checks/validations or just deps. to exist for them to be `Enabled`'d. In that case, those kind of feature-toggles can have their own specialised implementation of `Enable()` method whilst keeping the `magefile` isolated because the magefile is essentially just calling the `Enable()` method blindly over the instances of `featureToggleHandler` interface (the `availableFeatureToggles`) variable.